File: exercises/week05/exercise-dp-two-scala/src/facadepattern/TestFacade.scala
package facadepattern

object TestFacade {
  def main(args: Array[String]) {
    val scheduleServer: ScheduleServer = new ScheduleServerImpl
    val facadeServer: ScheduleServerFacade = new ScheduleServerFacade(scheduleServer)
    System.out.println("Start working......")
    facadeServer.startServer
    System.out.println("After work done.........")
    facadeServer.stopServer
  }
}

File: exercises/week05/exercise-dp-two/src/facadepattern/TestFacade.java
package facadepattern;

public class TestFacade {
    public static void main(String[] args) {
        ScheduleServer scheduleServer = new ScheduleServerImpl();
        ScheduleServerFacade facadeServer = new ScheduleServerFacade(scheduleServer);
        System.out.println("Start working......");
        facadeServer.startServer();
        System.out.println("After work done.........");
        facadeServer.stopServer();
    }
}